TCAT/Utilities/Endpoints.swift

```diff
@@ -18,54 +18,30 @@ enum NetworkType: String {
 }
 
 class Network {
+    
+    //
+    // Schemes
+    //
+    
+    // Production - Uses main production server for Network requests.
+    // Staging - Uses development server for Network requests, but will profile and archive to production.
 
     // MARK: Global Network Variables
 
-    /// Testing local servers for Network
-    // Change `networkType` to `.local` to work locally.
-    // Change `localIPAddress` to be the proper address
-
-    static let networkType: NetworkType = .debug
     static let apiVersion = "v1"
-
-    /// Used for local backend testing
-    static let localIPAddress = "10.132.0.68"
-    static let localSource = "http://\(localIPAddress):3000/api/\(apiVersion)/"
-
-    /// Test server used for development
-    static let debugIPAddress = "transit-testflight.cornellappdev.com"
-    static let debugSource = "https://\(debugIPAddress)/api/\(apiVersion)/"
-
-    /// Deployed server instance used for release
-    static let releaseIPAddress = "transit-backend.cornellappdev.com"
-    static let releaseSource = "https://\(releaseIPAddress)/api/\(apiVersion)/"
-
-    /// Network IP address being used for specified networkType
+    
     static var ipAddress: String {
-        #if RELEASE
-        if isTestFlight() {
-           return debugIPAddress
-        }
-        #endif
-        switch networkType {
-        case .local: return localIPAddress
-        case .debug: return debugIPAddress
-        case .release: return releaseIPAddress
+        // For local testing, use "http://\(localIPAddress):3000/api/\(apiVersion)/"
+        guard let baseURL = Bundle.main.object(forInfoDictionaryKey: "SERVER_URL") as? String else {
```

Comment:
if we have the two keys, here we can do
```
if #DEBUG 
let key = "LOCAL_SERVER_URL"
#else 
let key = "SERVER_URL"
#endif
guard let baseURL = Bundle.main.object(forInfoDictionaryKey: key) as? String else {...}
```

Reply:
<img width="578" alt="image" src="https://user-images.githubusercontent.com/8039928/54833557-b65e1280-4c94-11e9-8f3d-7839483a5c7a.png">

`SERVER_URL` changes based on the scheme!